sites/all/modules/unl_cas/unl_cas.module

```diff
@@ -125,6 +125,9 @@ function unl_cas_validate_ticket() {
   if ($auth) {
```

Comment:
I'm curious if you looked into getting ldap details passed via CAS (or if that is even being done right now)